ilastik_scripts/ilastik_startup.py

```diff
@@ -93,8 +94,14 @@ def _parse_internal_config(path: Union[str, os.PathLike]) -> Tuple[Sequence[str]
 
 def path_setup():
     if "--clean_paths" in sys.argv:
-        script_dir = pathlib.Path(__file__).parent
-        ilastik_root = script_dir.parent.parent
+        # this is only used in the windows binary - there python.exe is
+        # directly in ilastik_root
+        if platform.system != "Windows":
+            warnings.warn(
+                f"--clean_paths argument only supported on windows and should only be used in the binary distribution"
+            )
+        ilastik_root = pathlib.Path(sys.executable).parent
+        assert (ilastik_root / "ilastik.exe").exists()
         _clean_paths(ilastik_root)
```

Comment:
I think it's better to do nothing if preconditions for `--clean_paths` are not met:
```suggestion
        ilastik_root = pathlib.Path(sys.executable).parent
        if platform.system() == "Windows" and (ilastik_root / "ilastik.exe").is_file():
            _clean_paths(ilastik_root)
        else:
            warnings.warn("'--clean_paths' is supported only on Windows binary distributions")
```